cypress/e2e/cityinfo.cy.js

```diff
@@ -1,17 +1,13 @@
 describe("City Info", () => {
   beforeEach(() => {
-    cy.intercept(
-      "GET",
-      "https://dude-hows-my-city-be.herokuapp.com/api/v1/cities",
-      { fixture: "cities" }
-    );
-    cy.intercept(
-      "GET",
-      "https://dude-hows-my-city-be.herokuapp.com/api/v1/cities/1",
-      { fixture: "city" }
-    );
-    cy.visit("http://localhost:3000");
-    cy.get('[data-cy="city-name"]').first().click();
+    cy.intercept('GET','https://dude-hows-my-city-be.herokuapp.com/api/v1/cities', { fixture: 'cities' })
```

Comment:
Im sure I will be back here in the future for other projects. Nice to see how to intercept Posts